client/components/App.jsx

```diff
@@ -13,6 +13,8 @@ import {
   interviewMonthFolderName,
   prompts,
   suggestPrompt,
+  codestitchEmail,
```

Comment:
Is codestitch a hackreactor custom website?  I can't seem to sign up for an account and test it with my email and password on my laptop. I'll assume it works properly in the HiR lounge.